src/js/store/index.ts

```diff
@@ -0,0 +1,28 @@
+//@ts-ignore
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import { appStateReducer } from './appState/reducers';
+import { mapviewReducer } from './mapview/reducers';
+import { createLogger } from 'redux-logger';
+
+const rootReducer = combineReducers({
```

Comment:
@vaidotasp I'm glad we chatted about keeping `rootReducer` in `index.ts`! Since the boilerplate's root reducer is typically in its own file, I was thinking we could revisit this. I was thinking it could help maintain separation of concerns if the rootReducer has its own file. Just a suggestion - feel free to let me know what you think!